### PR TITLE
Moving setup.py to pyproject.toml

### LINKED ISSUE #341: Moving ``setup.py`` to ``pyproject.toml``
Removing `setup.py` in favor of the usage of `pyproject.toml`: from what I see in your codebase, there is no actual need to have a "dynamic installation" and thus you could align to the common standard of using a `pyproject.toml` file. In fact, you already have it, but you are only using it for tools configuration. This task would imply moving the `setup.py` information into the `pyproject.toml` format.

### PR DESCRIPTION
**Pull request by RobPasMue**
Closes #341

### CODE REVIEW

**Comment by RobPasMue** [comment]
Issues when building the docs - might be due to runner?

File: pyproject.toml [hunk 1]
```diff
@@ -1,13 +1,55 @@
 [build-system]
-requires = [ 
-    "setuptools>=59.2.0",
-    "wheel>=0.37.0",
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = "ansys-meshing-prime"
+version = "0.3.dev0"
```
Comment (RobPasMue) [inline pyproject.toml:7]:
```suggestion
version = "0.3.0.dev0"
```

**Comment by RobPasMue** [comment]
@ninad-kamat it seems that all workflow runs we executed on Friday failed to run successfully the "documentation" stage. There might be some issues with the runner. Could you please double check this? Thanks in advance! This affects also #342

**Comment by ninad-kamat** [comment]
@RobPasMue I will check on this issue once I am back from vacation

**Review by ninad-kamat — APPROVED** [review]

**Review by ninad-kamat — COMMENTED** [review]
@RobPasMue In the setup.py file, we added graphics images as package_data. I do not see the same in the pyproject.toml file. I am not familiar with flit. Could you explain how package data would work?

**Comment by RobPasMue** [comment]
@ninad-kamat you can check the artifacts for the wheel to confirm that the images are included =)

**Comment by ninad-kamat** [comment]
> @ninad-kamat you can check the artifacts for the wheel to confirm that the images are included =)

Will do. I will check once CI completes and if all looks good, will get this merged in today.

**Comment by RobPasMue** [comment]
> > @ninad-kamat you can check the artifacts for the wheel to confirm that the images are included =)
> 
> Will do. I will check once CI completes and if all looks good, will get this merged in today.

Awesome, thanks!

**Review by ninad-kamat — APPROVED** [review]

**Pull request merged**